src/components/GenerateGifForm.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import './GenerateGifForm.css';
+
+class GenerateGifForm extends Component {
+  constructor(props) {
+    super(props);
+    this.handleInputUpdate = this.handleInputUpdate.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(evt) {
+    evt.preventDefault();
+    this.props.handleGenerateGIF();
+    this.props.updateText('');
+    this.props.updateTextColor('');
```

Comment:
remove this as well, for the same reason

Reply:
Done